valohai_yaml/schema/step.yaml

```diff
@@ -22,6 +22,9 @@ properties:
   image:
     type: string
     description: The Docker image to use for this step.
+  custom-store:
+    type: string
+    description: The output data store name.
```

Comment:
Is it actually the name? Or is it a slug? Or an UUID?

Reply:
It might be both. I will change description